docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ExtractPixels.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.IO;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ExtractPixelsExample
+    {
+        public static void ExtractPixels()
+        {
+            var mlContext = new MLContext();
+
+            // Downloading a few images, and an images.tsv file, that contains a list of the files, from the dotnet/machinelearning/test/data/images/
+            // if you inspect the fileSystem, after running this line, the 
+            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+
+            // Preview of the content of the images.tsv file
+            //
+            // imagePath    imageType
+            // tomato.bmp	tomato
+            // banana.jpg   banana
+            // hotdog.jpg   hotdog
+            // tomato.jpg   tomato
+
+            var data = mlContext.Data.CreateTextLoader(new TextLoader.Arguments()
+            {
+                Column = new[]
+                {
+                        new TextLoader.Column("ImagePath", DataKind.TX, 0),
+                        new TextLoader.Column("Name", DataKind.TX, 1),
+                    }
+            }).Read(imagesDataFile);
+
+            var imagesFolder = Path.GetDirectoryName(imagesDataFile);
+            // Image loading pipeline. 
+            var pipeline = mlContext.Transforms.LoadImages(imagesFolder, ("ImageObject", "ImagePath"))
+                          .Append(mlContext.Transforms.Resize("ImageObject",imageWidth: 100 , imageHeight: 100 ))
+                          .Append(mlContext.Transforms.ExtractPixels("Pixels", "ImageObject"));
```

Comment:
@Anipik: We could make a good benchmark for the image processing pipeline. 
I'd recommend using the Dog Breeds vs. Fruits dataset which we used in [NimbusML for its image examples](https://docs.microsoft.com/en-us/nimbusml/tutorials/b_f-image-processing-clustering). We currently host this dataset in our CDN for NimbusML.

In Python, the dataset / image loader looks like:
```Python
# Load image summary data from github
url = "https://express-tlcresources.azureedge.net/datasets/DogBreedsVsFruits/DogFruitWiki.SHUF.117KB.735-rows.tsv"
df_train = pd.read_csv(url, sep = "\t", nrows = 100)
df_train['ImagePath_full'] = "https://express-tlcresources.azureedge.net/datasets/DogBreedsVsFruits/" + \
                         df_train['ImagePath']
... load images
```

Purpose of the dataset is for example code & includes ~775 images of dogs & fruit:
![image](https://user-images.githubusercontent.com/4080826/52151638-ab89e880-2628-11e9-9df5-2b060875e56e.png)
![image](https://user-images.githubusercontent.com/4080826/52151656-b5abe700-2628-11e9-8f5a-483b3ddf20e5.png)

The Dog Breeds vs. Fruits would also be nice for our samples repo. 

/cc @CESARDELATORRE

Reply:
@jormont, maybe log an issue about it. It might get forgotten here as a comment. 

---
In reply to: [253212682](https://github.com/dotnet/machinelearning/pull/2372#discussion_r253212682) [](ancestors = 253212682)

Reply:
@sfilipi -- good idea:
* Benchmark: https://github.com/dotnet/machinelearning/issues/2379
* Sample: https://github.com/dotnet/machinelearning-samples/issues/225